server/liveblog/themes/themes_assets/classic/main.js

```diff
@@ -5,20 +5,30 @@
     function TimelineCtrl($interval, PagesManager, blogsService, config, $anchorScroll, $timeout, Permalink, transformBlog) {
 
         var POSTS_PER_PAGE = config.settings.postsPerPage;
+        var STICKY_POSTS_PER_PAGE = 100;
```

Comment:
I meant API pagination. What to do if there are more than 100 sticky posts for instance?